File: leku/src/main/java/com/schibstedspain/leku/tracker/TrackEvents.java
package com.schibstedspain.leku.tracker;

public enum TrackEvents {
  googleApiConnectionFailed("Connection Failed"),
  startVoiceRecognitionActivityFailed("Start Voice Recognition Activity Failed"),
  didLoadLocationPicker("Location Picker"),
  didSearchLocations("Click on search for locations"),
  didLocalizeMe("Click on localize me"),
  didLocalizeByPoi("Long click on map"),
  simpleDidLocalizeByPoi("Click on map"),
  simpleDidLocalizeByLekuPoi("Click on POI"),
  RESULT_OK("Return location"),
  CANCEL("Return without location");

  private String eventName;

  TrackEvents(String eventName) {
    this.eventName = eventName;
  }

  public String getEventName() {
    return eventName;
  }
}

File: leku/src/main/java/com/schibstedspain/leku/tracker/TrackEvents.kt
package com.schibstedspain.leku.tracker

enum class TrackEvents(val eventName: String) {
    GOOGLE_API_CONNECTION_FAILED("Connection Failed"),
    START_VOICE_RECOGNITION_ACTIVITY_FAILED("Start Voice Recognition Activity Failed"),
    ON_LOAD_LOCATION_PICKER("Location Picker"),
    ON_SEARCH_LOCATIONS("Click on search for locations"),
    ON_LOCALIZED_ME("Click on localize me"),
    ON_LOCALIZED_BY_POI("Long click on map"),
    SIMPLE_ON_LOCALIZE_BY_POI("Click on map"),
    SIMPLE_ON_LOCALIZE_BY_LEKU_POI("Click on POI"),
    RESULT_OK("Return location"),
    CANCEL("Return without location")
}
